packages/component/src/Activity/StackedLayout.js

```diff
@@ -88,6 +88,14 @@ const StackedLayout = ({ activity, avatarInitials, children, timestampClassName
   const [{ botAvatarInitials, bubbleNubSize, bubbleFromUserNubSize, userAvatarInitials }] = useStyleOptions();
   const [{ stackedLayout: stackedLayoutStyleSet }] = useStyleSet();
 
+  const botAriaLabel = useLocalize('Bot said something', avatarInitials, plainText);
+  const userAriaLabel = useLocalize('User said something', avatarInitials, plainText);
+  const botRoleLabel = useLocalize('BotSent');
```

Comment:
`botRoleLabel` is already defined on line 115

Reply:
Fixed, bad merge.